proxy/proxy.go

```diff
@@ -69,6 +71,7 @@ func New(cfg *config.Config) (*Proxy, error) {
 		currentOptions: config.NewAtomicOptions(),
 		currentRouter:  atomicutil.NewValue(httputil.NewRouter()),
 	}
+	p.webauthn = webauthn.New(p.getWebauthnState)
```

Comment:
I think you can put the assignment in the above struct assignement? Maybe there's a reason I'm missing though.

Reply:
It references itself (`p.getWebauthnState`), so it can't be done in the struct assignment